includes/class-wc-order-factory.php

```diff
@@ -100,8 +100,12 @@ public static function get_order_item( $item_id = 0 ) {
 				case 'tax' :
 					$classname = 'WC_Order_Item_Tax';
 				break;
+				default :
+					$classname = apply_filters( 'woocommerce_get_order_item_classname', $classname, $item_type, $item_id );
+				break;
 			}
-			if ( $classname ) {
+
+			if ( class_exists( $classname ) ) {
```

Comment:
I'd prefer an empty check - checking `class_exists` on an empty string is still not desirable.